jobs/gorouter/templates/gorouter.yml.erb

```diff
@@ -336,6 +336,7 @@ params['logging'] = {
   'disable_log_forwarded_for' => p('router.disable_log_forwarded_for'),
   'disable_log_source_ip' => p('router.disable_log_source_ip'),
   'redact_query_params' => p('router.redact_query_parameters'),
+  'attempts_details' => p('router.log_attempts_details'),
```

Comment:
This needs adaptation to the new naming in https://github.com/cloudfoundry/gorouter/pull/346.

-> https://github.com/cloudfoundry/gorouter/pull/346/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R151